ci-r/Dockerfile

```diff
@@ -39,6 +39,23 @@ RUN apt-get update \
 RUN apt-get update \
         && install.r digest docopt testthat devtools roxygen2
         
-COPY buildTileDB.sh /usr/local/bin
+# Release version number of TileDB to install.
+ARG version=1.7.2
+
+# Install TileDB
+RUN wget -P /home/tiledb https://github.com/TileDB-Inc/TileDB/archive/${version}.tar.gz \
+    && tar xzf /home/tiledb/${version}.tar.gz -C /home/tiledb \
+    && rm /home/tiledb/${version}.tar.gz \
+    && cd /home/tiledb/TileDB-${version} \
+    && mkdir build \
+    && cd build \
+    && cmake .. -DTILEDB_S3=ON -DTILEDB_HDFS=OFF -DCMAKE_BUILD_TYPE=DEBUG -DTILEDB_SERIALIZATION=ON \
+    && make -j$(nproc) \
+    && make install-tiledb \
+    && rm -rf /home/tiledb/TileDB-${version}
+
+RUN echo "/usr/local/lib" >> /etc/ld.so.conf.d/usr-local.conf
+RUN echo "/usr/local/lib64" >> /etc/ld.so.conf.d/usr-local.conf
+RUN ldconfig /usr/local/lib
```

Comment:
That should probably be one `RUN` command, and the `ld.so.conf.d` entry is not needed on Ubuntu.

Reply:
Removed/simplified